src/app/client/src/app/plugins/location/components/location-selection/location-selection.component.ts

```diff
@@ -56,10 +58,18 @@ export class LocationSelectionComponent implements OnInit, OnDestroy, AfterViewI
 
   ngOnInit() {
     this.popupControlService.changePopupStatus(false);
+    /**
+      * @description - Here we enable/disable the location popup based on the isvisible value returned from the form config request
+    */
+    this.popupControlService.getOnboardingData().subscribe((formResponsedata)=>{
+      this.isLocationEnabled= formResponsedata.locationPopup? formResponsedata.locationPopup.isVisible : true;
+    })
     this.sbFormLocationSelectionDelegate.init(this.deviceProfile, this.showModal, this.isStepper )
       .catch(() => {
         this.closeModal();
-        this.toasterService.error(this.resourceService.messages.fmsg.m0049);
+        if(this.isLocationEnabled === true){
```

Comment:
@mithun30052001 refactor this if(this.isLocationEnabled === true){

Reply:
Refactor done @princegupta1131